src/Entity/Feedback.php

```diff
@@ -114,6 +114,14 @@ public function setComment($comment) {
     $this->set('comment', $comment);
   }
 
+  public function getAccessibility() {
```

Comment:
Naming this so it's clear it's about feedback type would make it easier to get the use.

Reply:
Hmm, yes the naming scheme would make more sense if 'is' and 'set' prefixes would be used instead of 'get' and 'set', since this is a boolean value.